README.md

```
@@ -858,3 +858,94 @@ MIT
+| **HTML** | Executive security report |
+| **Evidence Pack** | Portable audit bundle for compliance |
+
+### What is the Opus Pipeline?
+
+The Opus Pipeline is a **three-agent adversarial analysis** using Anthropic's Opus 4.6 model:
+1. **Analyzer Agent** — Deep analysis of findings
+2. **Adversarial Agent** — Attack simulation and impact assessment
+3. **Reviewer Agent** — Final recommendations and mitigation strategies
+
+**Usage:**
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)